pkg/proxy/handlers/spacelister_get.go

```diff
@@ -119,7 +150,7 @@ func filterUserSpaceBinding(username string, allSpaceBindings []toolchainv1alpha
 
 // generateWorkspaceBindings generates the bindings list starting from the spacebindings found on a given space resource and an all parent spaces.
 // The Bindings list  has the available actions for each entry in the list.
-func generateWorkspaceBindings(space *toolchainv1alpha1.Space, spaceBindings []toolchainv1alpha1.SpaceBinding) ([]toolchainv1alpha1.Binding, error) {
+func generateWorkspaceBindings(space *toolchainv1alpha1.Space, spaceBindings []toolchainv1alpha1.SpaceBinding, spacebindingRequests []toolchainv1alpha1.SpaceBindingRequest) ([]toolchainv1alpha1.Binding, error) {
 	var bindings []toolchainv1alpha1.Binding
```

Comment:
something I noticed here, though it's not an issue with your patch per se.  `bindings` should be at most of size `len(spaceBindings) + len(spacebindingRequests)`, so maybe we should do this allocation once?

Don't need to do this change in this PR, but it's something to consider.
```suggestion
	bindings := make([]toolchainv1alpha1.Binding, 0, len(spaceBindings) + len(spacebindingRequests))
```

Reply:
makes sense indeed. Thanks!